elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/AggregationDataStore.java

```diff
@@ -34,7 +37,8 @@
     /**
      * These are the classes the Aggregation Store manages.
      */
-    private static final Class[] AGGREGATION_STORE_CLASSES = { FromTable.class, FromSubquery.class };
+    private static final Class[] AGGREGATION_STORE_CLASSES = {
+            Subselect.class, Table.class, FromTable.class, FromSubquery.class };
```

Comment:
I don't think we want AggregationStore to manage Entities at all.  In that case - we should remove Table and Subselect.

Reply:
Currently the **view**s are using `@Table` and `@Subselect` to distinguish themselves from **analyticView**s. Those models need to be managed by AggregationStore.